app/aws/s3.py

```diff
@@ -189,3 +191,33 @@ def generate_presigned_url(bucket_name: str, object_key: str, expiration: int =
         return ""
 
     return response
+
+
+def stream_to_s3(bucket_name, object_key, copy_command, cursor):
+    """
+    Stream data from PostgreSQL COPY command directly to S3.
+
+    :param bucket_name: S3 bucket name
+    :param object_key: S3 object key
+    :param copy_command: PostgreSQL COPY command
+    :param cursor: Database cursor
+    """
+    s3_client = client("s3", current_app.config["AWS_REGION"])
+    config = TransferConfig(multipart_threshold=1024 * 25, max_concurrency=10)
```

Comment:
can we pass these two numbers in as variables in the func?
is it worth putting this in config

Reply:
I've moved them to be args to that function. We could move them to config if it turns out that we want different values for different environments. Currently these are just guesses at values to test out.